packages/convict/MIGRATING_FROM_CONVICT_5_TO_6.md

```diff
@@ -32,7 +32,7 @@ convict.addFormats(convict_format_with_moment)
 
 // Use this only if you have a .json configuration file in JSON5 format
 // (i.e. with comments, etc.).
-convict.addParser({extension: 'json', parse: JSON.parse})
+convict.addParser({extension: 'json', parse: JSON5.parse})
```

Comment:
@madarche are you sure this isn’t a typo? Using the native `JSON.parse` on json5 would throw an error here, and i believe the current `json5` import from line 25 is currently unused.

Reply:
@pdehaan you're right! I've mistakenly closed this PR: I've thought its aim was just to rename `json5`→`JSON5`. And @xamgore sorry!